internal/configs/transportserver_test.go

```diff
@@ -62,6 +62,164 @@ func TestTransportServerExString(t *testing.T) {
 	}
 }
 
+func TestGenerateTransportServerConfigForTCPSnippets(t *testing.T) {
```

Comment:
input and output for TestGenerateTransportServerConfigForTCPSnippets 
 and TestGenerateTransportServerConfigForTCPSnippetsNotEnabled are almost the same. 
 
 is there a way to reduce duplication ?

Reply:
I think in the case of two tests, the cost of the increased complexity doesn't help. If there were several tests, it would be a great idea.